packages/sdk-browser/src/sdk-client.ts

```diff
@@ -375,7 +434,7 @@ export class TurnkeyPasskeyClient extends TurnkeyBrowserClient {
     expirationSeconds: string = DEFAULT_SESSION_EXPIRATION,
     organizationId?: string
   ): Promise<ReadWriteSession> => {
-    const localStorageUser = await getStorageValue(StorageKeys.CurrentUser);
+    const localStorageUser = await getStorageValue(StorageKeys.UserSession);
     userId = userId ?? localStorageUser?.userId;
```

Comment:
@andrewkmin should we make userId optional given that it could comes from the local storage similar to organizationId?